### PR TITLE
Cypress/E2E: Fix tests related to email verification

### PR DESCRIPTION
**Pull request by saturninoabril**
#### Summary
Failed tests were mostly due to recent change on email template and error message on server

#### Ticket Link
none, failed on daily Cypress run

#### Screenshots
Screen Shot 2021-04-21 at 8 30 07 AM
Screen Shot 2021-04-21 at 8 30 07 AM
Screen Shot 2021-04-21 at 8 30 07 AM
Screen Shot 2021-04-21 at 8 30 07 AM
Screen Shot 2021-04-21 at 8 30 07 AM

```release-note
NONE
```

### CODE REVIEW

File: e2e/cypress/integration/enterprise/cloud/onboarding/login_page_link_account_creation_spec.js [hunk 1]
```diff
@@ -7,7 +7,7 @@
 // - Use element ID when selecting an element. Create one if none.
 // ***************************************************************
 
-// Group: @enterprise @onboarding
+// Group: @enterprise @onboarding @cloud_only
```
Comment (saturninoabril) [inline e2e/cypress/integration/enterprise/cloud/onboarding/login_page_link_account_creation_spec.js:10]: Added `@cloud_only` so we can filter this out specifically for cloud test run.

**Review by josephbaylon — APPROVED** [review]
LGTM. Most tests passed locally except for `use_team_invite_link_to_sign_up_spec.js` but the failure is expected since it's failing on the same verification in unstable tests report.

**Review by prapti — APPROVED** [review]
LGTM. Thanks Saturn!

**Comment by saturninoabril** [comment]
Now updated. `MM-T398` should now work for TE, EE and Cloud.

File: e2e/cypress/integration/onboarding/use_team_invite_link_to_sign_up_spec.js [hunk 2]
```diff
@@ -98,12 +104,12 @@ describe('Onboarding', () => {
             // # Check that 'Email Verified' text should be visible, email is pre-filled, and password field is focused, then login
             cy.findByText('Email Verified', {timeout: TIMEOUTS.HALF_MIN}).should('be.visible');
             cy.get('#loginId').should('have.value', email);
-            cy.get('#loginPassword').should('be.focused').type(password);
```
Comment (saturninoabril) [inline e2e/cypress/integration/onboarding/use_team_invite_link_to_sign_up_spec.js:101]: Focused on password field is as expected in manual check but it's not reliable to be checked in Cypress (maybe due to some re-render happening in a page) and so, I intentionally remove it in test.

**Review by josephbaylon — APPROVED** [review]
LGTM. Retested `use_team_invite_link_to_sign_up_spec.js` and now passes on different licenses.

**Review by furqanmlk — APPROVED** [review]
Locally Passed
image

image

image

image

image

**Pull request merged**